### DISCUSSION TITLE
Customizable Font Family

### DISCUSSION THREAD

**Discussion by topada** [dicebear/dicebear#119] [Ideas]
Hi there,
I'll be using diceBear initials avatars soon, and I'd love to customize the font family. 
Surely, this has some limitations but a selection of google fonts would be highly appreciated.

`https://avatars.dicebear.com/api/initials/:seed.svg?fontFamily=RobotoMono`

**Comment by m5o** [comment]
Google Fonts would be an awesome feature, if this is not possible yet there is a safe `font-family` list:
https://www.w3schools.com/cssref/css_websafe_fonts.asp

**Comment by FlorianKoerner** [comment]
I like the idea too. But with Google Webfonts I see a GDPR problem in the context of the HTTP API. Maybe the web safe fonts are a good compromise.

But we also need a way to convert the fonts to paths here in the future. That should be considered as well.

**Comment by FlorianKoerner** [comment]
Will be available in version 5.0.
